custom_components/better_thermostat/climate.py

```diff
@@ -661,7 +661,7 @@ async def startup(self):
             await asyncio.sleep(10)
             self._available = True
             self.startup_running = False
-            self.async_update_ha_state()
+            await self.async_update_ha_state()
```

Comment:
Think this isn't an await function

Reply:
I got an error on the devcontainer, that's why i added it. But i will remove it.

/config/custom_components/better_thermostat/climate.py:664: RuntimeWarning: coroutine 'Entity.async_update_ha_state' was never awaited
  self.async_update_ha_state()